Resources/Prototypes/Entities/Mobs/Customization/Markings/cat_parts.yml

```diff
@@ -2,7 +2,7 @@
   id: CatEars
   bodyPart: HeadTop
   markingCategory: HeadTop
-  speciesRestriction: [Human, Felinid]
+  speciesRestriction: []
```

Comment:
does this make them unusable or is this going to make them valid for every species?

Reply:
alright, it's just a non null empty list

Reply:
It makes them unusuable, because the empty list is not considered null. It checks for null, rather than count of items to determine whether to grant it to everyone. I did not completely remove the marking itself out of concerns of backwards compatibility w.r.t. saved player characters.

I wasn't able to reproduce it, but when I was adding and removing my own markings, I got to a point where I wasn't able to log in because my client crashed, until I added back the old marking I made, which I had saved on a character. Just being cautious.